lib/active_shipping/carriers/fedex.rb

```diff
@@ -153,8 +161,131 @@ def find_tracking_info(tracking_number, options = {})
       parse_tracking_response(xml, options)
     end
 
+
+    # Get Shipping labels
+    # Caveats:
+    #  - Only supports singlular packages
+    #
+    def create_shipment(origin, destination, packages, options = {})
+      options = @options.update(options)
+      packages = Array(packages)
+
+      begin
+        request = build_shipment_request(origin, destination, packages, options)
+        logger.debug(request) if logger
+
+        logger.debug(confirm_response) if logger
+
+        response = commit(save_request(request), options[:test])
```

Comment:
looking at the other `commit` calls, should the last arg be `options[:test] || false`?